lib/generate-stylesheet.js

```diff
@@ -7,24 +7,29 @@ rework.comments = require('rework-comments');
 const CONSTANTS = require('./_constants');
 
 
-const styles = fs.readFileSync(__dirname + '/style.css', 'utf8');
-const reworkCSS = rework(styles)
-  // DEV: Adds a .custom selector to the beginning of all CSS selectors
-  .use(rework.namespace({
-    selector: '.' + CONSTANTS.CLASS_NAMESPACE,
-    root: 'body',
-    namespaceBody: false,
-  }))
-  // DEV: Adds !important flag to all CSS rules
-  .use(rework.important())
-  // DEV: Removes all CSS comments
-  .use(rework.comments)
-  .toString();
-try {
-  fs.mkdirSync(__dirname + '/../build');
-} catch (e) {
-  // If the build directory exists absorb the error
-}
-// DEV: Write the CSS minified to the temporary CSS file
-fs.writeFileSync(__dirname + '/../build/rework.css', (reworkCSS));
-console.info('Rework CSS Generated'); // eslint-disable-line
+const parseCSSFile = (fileName, rootNamespace) => {
+  const styles = fs.readFileSync(__dirname + '/' + fileName + '.css', 'utf8');
+  const reworkCSS = rework(styles)
+    // DEV: Adds a .custom selector to the beginning of all CSS selectors
+    .use(rework.namespace({
+      selector: '.' + CONSTANTS.CLASS_NAMESPACE,
+      root: rootNamespace,
+      namespaceBody: false,
+    }))
+    // DEV: Adds !important flag to all CSS rules
+    .use(rework.important())
+    // DEV: Removes all CSS comments
+    .use(rework.comments)
+    .toString();
+  try {
+    fs.mkdirSync(__dirname + '/../build');
+  } catch (e) {
+    // If the build directory exists absorb the error
+  }
+  // DEV: Write the CSS minified to the temporary CSS file
+  fs.writeFileSync(__dirname + '/../build/' + fileName + '.css', (reworkCSS));
+  console.info('Rework CSS Generated'); // eslint-disable-line
+};
+
+parseCSSFile('highlight', 'body');
```

Comment:
It used to be because we had both the highlight and full CSS being applied to the page at the same time. As a result, we had to use additional CSS selectors to say what to show. Now that we are dynamically creating the CSS to be a highlight or a full render, it's no longer necessary